airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/ad_group_ad_report.json

```diff
@@ -0,0 +1,767 @@
+{
+  "name": "ad_group_ad_report",
+  "supported_sync_modes": ["full_refresh", "incremental"],
+  "source_defined_cursor": true,
+  "default_cursor_field": ["date"],
+  "json_schema": {
+    "type": "object",
+    "title": "Ad Group Ad Report",
+    "properties": {
+      "accent_color": {
+        "description": "AccentColor",
+        "type": ["null", "string"],
+        "field": "ad_group_ad.ad.legacy_responsive_display_ad.accent_color"
```

Comment:
I am concerned about how we are using the `field` value in a few ways: 
1. We are diverging from the API's official naming of these fields, which means a user must look up both the Ads documentation and the Airbyte GAds documentation to understand which field is which
2. It is probably arduous and prone to error to input this by hand. We will want to support 10,20,30 reports on Google Ads in the near future, this will become very difficult to keep in sync across all the reports and will require lots of time to input. 
3. Non-report endpoints (e.g: [AdGroup](https://developers.google.com/google-ads/api/reference/rpc/v7/AdGroup)) don't have the same concept of fields, so the way we are extracting fields from those responses will not be the same

Why not keep using the official Google field names?